desktop/npm-helper.js

```diff
@@ -179,6 +179,14 @@ const commands = {
     help: 'Undiff log send',
     code: undiff,
   },
+  'generate-font-project': {
```

Comment:
New font helpers. I tried to go entirely using a local thing but in the end it was too complicated. So instead we now generate our own iconmoon project locally and don't parse css anymore so its more controlled. I also got cecile to rename the svgs so we get the grid sizes so this whole thing can be automated mostly (with in-between steps submitting things to icomoon)